config/crd/crd.projectcalico.org_felixconfigurations.yaml

```diff
@@ -342,11 +342,7 @@ spec:
                   in ipvs mode. Felix uses these ranges to separate host and workload
                   traffic. [Default: 30000:32767].'
                 items:
-                  anyOf:
```

Comment:
Hm, these changes seem wrong.

Reply:
It seems our customised `controller-gen` binary needs to be updated because `numorstring` package has been moved to `github.com/projectcalico/api/pkg/lib/numorstring` 
https://github.com/caseydavenport/controller-tools/blob/support-floats/pkg/crd/known_types.go#L55

@caseydavenport Could you help to update the code and release a new version?